shared/search/user-pane/non-user.render.desktop.js

```diff
@@ -0,0 +1,125 @@
+/* @flow */
+
+import React, {Component} from 'react'
+import {Avatar, Box, Icon, Text} from '../../common-adapters'
+import {globalColors, globalStyles, globalMargins} from '../../styles/style-guide'
+import {capitalize} from 'lodash'
+import type {Props} from './non-user.render'
+import type {Props as IconProps} from '../../common-adapters/icon'
+
+import electron from 'electron'
+const shell = electron.shell || electron.remote.shell
+
+export default class Render extends Component<void, Props, void> {
+  _onClickAvatar () {
+    shell.openExternal(this.props.profileURL)
+  }
+
+  _iconNameForService (serviceName: string): IconProps.type {
+    return {
+      'twitter': 'twitter-logo-24',
+      'github': 'github-logo-24',
+      'reddit': 'reddit-logo-24',
+      'pgp': 'icon-pgp-key-24',
+      'coinbase': 'coinbase-logo-24'
+    }[serviceName]
```

Comment:
i basically added that here: https://github.com/keybase/client/pull/3056/files#diff-8dc4be3d54fcd1d571feade868b7da84R40